scripts/deploy-prometheus.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
+
+# (c) Copyright [2021-2024] Open Text.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A script that will deploy the prometheus service monitor and secret. It assumes the prometheus operator is deployed.
+
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+REPO_DIR=$(dirname $SCRIPT_DIR)
+LABEL=''
+ACTION=''
+NAMESPACE=''
+USERNAME=''
+PASSWORD=''
+VDB_NAME=''
+INTERVAL='5s'
+
+function usage() {
+    echo "usage: $(basename $0) [-n <namespace>] [-l <label>] [-a <action>] [-u <username>] [-p <password>] [-d <vdbname>] [-i <interval>]"
+    echo
+    echo "Options:"
+    echo "  -n <namespace>  The namespace used for prometheus service."
+    echo "  -l <label>      The label monitored by prometheus service."
+    echo "  -a <action>     The action to run in this script, deploy or undeploy."
+    echo "  -u <username>   The database username, should have access to the Vertica server metrics."
+    echo "  -p <password>   The database user password."
+    echo "  -d <vdbname>    The k8s CRD database name."
```

Comment:
```suggestion
    echo "  -d <vdbname>    The CR  name."
```
It is not the database name.

Reply:
Updated in https://github.com/vertica/vertica-kubernetes/pull/1013/commits/c82437dac9079d1f5185531766cca2870c24ffe3